### PR TITLE
FF_RACEPIT Beeper resource use C03

### PR DESCRIPTION
**Pull request by nerdCopter**
* fix FuriousFPV RacePit beeper resource #463 , #464 
* waiting on test/verification

### CODE REVIEW

**Comment by Quick-Flash** [comment]
will approve when verified

**Comment by nerdCopter** [comment]
this needs approval. 3rd party found the correct resource. no need for further testing, no one testing.

**Review by Quick-Flash — APPROVED** [review]

**Pull request merged**
